site/source/user-manual/dashboard-overview.rst

```diff
@@ -47,6 +74,10 @@ The Support section provides links to frequently asked questions as well as Star
 Power
```

Comment:
Please add remaining power options - Shutdown and System Rebuild

Reply:
They are there already?